pkgs/data/themes/arc/default.nix

```diff
@@ -20,8 +20,9 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "jnsh";
     repo = pname;
-    rev = version;
-    sha256 = "sha256-BNJirtBtdWsIzQfsJsZzg1zFbJEzZPq1j2qZ+1QjRH8=";
+    rev =  version;
+    sha256 = "BNJirtBtdWsIzQfsJsZzg1zFbJEzZPq1j2qZ+1QjRH8=";
+    fetchSubmodules = false;
```

Comment:
`fetchSubmodules` is false by default:

https://github.com/NixOS/nixpkgs/blob/4e211c1b09e70a1c4d0c164ea51497fd31c30c0b/pkgs/build-support/fetchgithub/default.nix#L4